.pre-commit-config.yaml

```diff
@@ -12,7 +12,7 @@ repos:
     - id: check-yaml
     - id: debug-statements
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.4.9
+    rev: v0.4.10
```

Comment:
Its job of pre-commit bot, which will be chery-picking until 6.13, which you're not doing here

Reply:
Ok removed this file